vocabulary-draft/continuous-deployment-pipeline-events.md

```diff
@@ -0,0 +1,26 @@
+# Continuous Deployment Pipelines Events 
```

Comment:
I would drop the "continuous" part here

Reply:
@afrittoli any reasons behind that? I think that I see a point of just using these events for Deployments in a non-continuous way, but I would keep continuous here to relate to the business value.